### DISCUSSION TITLE
Wiki Error making it unreadable

### DISCUSSION THREAD

**Discussion by ghost** [community/community#23144] [New to GitHub]
<p>Hello. My GitHub wiki is showing like:<br>
<code> &lt;h2&gt; &lt;a id="user-content-upbeatversion" class="anchor" href="#upbeatversion" aria-hidden="true"&gt;&lt;svg class="octicon octicon-link" viewbox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"&gt;&lt;path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"&gt;&lt;/path&gt;&lt;/svg&gt;&lt;/a&gt;upbeat.version()&lt;/h2&gt; &lt;p&gt;Returns the UpBeat Python Module version installed.&lt;br&gt;Type: &lt;code&gt;str&lt;/code&gt;.&lt;/p&gt; &lt;h2&gt; &lt;a id="user-content-upbeatstatslisteners" class="anchor" href="#upbeatstatslisteners" aria-hidden="true"&gt;&lt;svg class="octicon octicon-link" viewbox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"&gt;&lt;path fill-rule="evenodd" d="M7.775 3.275a.75.75 0 001.06 1.06l1.25-1.25a2 2 0 112.83 2.83l-2.5 2.5a2 2 0 01-2.83 0 .75.75 0 00-1.06 1.06 3.5 3.5 0 004.95 0l2.5-2.5a3.5 3.5 0 00-4.95-4.95l-1.25 1.25zm-4.69 9.64a2 2 0 010-2.83l2.5-2.5a2 2 0 012.83 0 .75.75 0 001.06-1.06 3.5 3.5 0 00-4.95 0l-2.5 2.5a3.5 3.5 0 004.95 4.95l1.25-1.25a.75.75 0 00-1.06-1.06l-1.25 1.25a2 2 0 01-2.83 0z"&gt;&lt;/path&gt;&lt;/svg&gt;&lt;/a&gt;upbeat.stats.listeners()&lt;/h2&gt; &lt;p&gt;Returns the listener count.&lt;br&gt;Type: &lt;code&gt;str&lt;/code&gt;.&lt;/p&gt; &lt;h2&gt; &lt;a id="user-content-upbeatstatsupdated" class="anchor" href="#upbeatstatsupdated" aria-hidden="true"&gt;&lt;svg class="octicon octicon-link" viewbox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"&gt;</code><br>
How do I stop this.</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>I’m so sorry about this! This is broken for many people right now and we have a fix being deployed as priority. Hopefully it’ll be working properly for you soon.</p>

**Comment by shiftkey** [comment, marked as answer]
<p>:wave: the fix has been deployed, so hopefully this is resolved for you <a class="mention" href="/danieldot">@DanielDot</a>!</p>
